test/cert-ceremonies/generate.go

```diff
@@ -73,6 +73,12 @@ func genCert(path string) error {
 }
 
 func main() {
+	// If one of the output files already exists, assume this ran once
+	// already for the container and don't re-run.
+	if _, err := os.Stat("/tmp/root-signing-pub-rsa.pem"); err == nil {
```

Comment:
[os.IsNotExist](https://golang.org/pkg/os/#IsNotExist) would be a better guard for the assertion being made. If an error is encountered in the `os` package it will return a `*PathError`; [os.IsNotExist](https://golang.org/pkg/os/#IsNotExist) returns a boolean based on whether the underlying error is `ErrNotExist`.